example/settings.py

```diff
@@ -130,6 +130,13 @@
         "allowed_values": ["Policy", "Claim", "IVR", "Purchase", "Pre Inspection", "CX 360", "Reporting"],
     },
 }
-
 TE_ROWLIMIT = 1000
 TE_BASEPATH = "/"
+# GDAL_LIBRARY_PATH="/opt/homebrew/Cellar/gdal/3.4.1_1/lib -lgdal"
+GDAL_LIBRARY_PATH = '/opt/homebrew/opt/gdal/lib/libgdal.dylib'
+GEOS_LIBRARY_PATH = '/opt/homebrew/opt/geos/lib/libgeos_c.dylib'
+
+WKPDFGEN_SERVICE_URL="https://wkpdfgen.ackodev.com/"
+WKPDFGEN_ASSET_URL="https://wkpdfgen-uat.ackoassets.com/pdfgen/templates"
+USER_SERVICE_URL="http://entity-service.internal.ackodev.com/api/user/"
```

Comment:
I don't think these have anything to do with this PR. Maybe this is for your local setup?